src/data/markdown/docs/02 javascript api/06 k6-execution.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "k6/execution"
+excerpt: "Get information about the current test's execution state."
+---
+
+k6 v0.34.0 introduced the capability for getting information about the current test execution state, from inside of the test's script.
+
+It can be used by importing the `k6/execution` module.
+
+<div class="code-group" data-props='{"labels": [], "lineNumbers": [true]}'>
+
+```javascript
+import exec from 'k6/execution';
+
+var options = {
+	scenarios: {
+		'myscenario': { // this will be the returned name
+			executor: 'shared-iterations',
+			maxDuration: '30m'
+		}
+	}
+}
+
+export default function() {
+	console.log(exec.scenario.name) // myscenario 
```

Comment:
Two-space indents instead of tabs ;)

I would also change the other examples.

```suggestion
  scenarios: {
    'myscenario': { // this will be the returned name
      executor: 'shared-iterations',
      maxDuration: '30m'
    }
  }
}

export default function() {
  console.log(exec.scenario.name) // myscenario 
```